arm-network/2017-06-01/examples/SubnetCreate.json

```diff
@@ -0,0 +1,32 @@
+{
+  "parameters": {

```

Comment:
parameters are not properly specified in the example, please add "subnetName" to the parameters and "subnetParameters" with its properties "name" and "addessPrefix" inside.